### PR TITLE
fix not constructor and add unit test for parseFunctionData

### PR DESCRIPTION
**Pull request by PKUzjw**
# PR Details

## Description

1. fix the bug --- not a constructor
2. add unit test for parseFunctiondata
## Related Issue

None

## How Has This Been Tested

use unit test
## Types of changes

- [x]  add unit test

## Checklist

- [x]   My code follows the code style of this project.
- [x]   All new and existing tests passed.

### CODE REVIEW

File: test/test_token.js [hunk 1]
```diff
@@ -262,6 +276,7 @@ describe('test split token', function () {
     // Only sendToken function needs attachment
     let function_index = constants.SPLIT_FUNCIDX;
     let contract_tx = tra.buildExecuteContractTx(public_key, contract_id, function_index, function_data, timestamp, attachment);
+    console.log('contract_tx:  ', contract_tx);
```
Comment (Haynar216) [inline test/test_token.js]: delete console.log

File: test/test_token.js [hunk 2]
```diff
@@ -273,6 +288,12 @@ describe('test split token', function () {
     let bytes = tra.toBytes();
     let signature = acc.getSignature(bytes);
     let send_tx = tra.toJsonForSendingTx(signature);
+    let parse_function_data = convert.parseFunctionData(send_tx['functionData']);
+    console.log('result: ', parse_function_data);
```
Comment (Haynar216) [inline test/test_token.js]: delete console.log

**Comment by SheldonYS** [comment]
The update of `package.json` should be committed.

File: test/test_token.js [hunk 3]
```diff
@@ -73,7 +73,12 @@ describe('test create token', function () {
     let bytes = tra.toBytes();
     let signature = acc.getSignature(bytes);
     let send_tx = tra.toJsonForSendingTx(signature);
-
+    let create_parse_function_data = convert.parseFunctionData(send_tx['initData']);
+    let create_original_data = BigNumber(contractTx['initData']['amount']).multipliedBy(contractTx['initData']['unity']);
+    it('unit test for parseFunctionData when destroy token', function() {
```
Comment (Haynar216) [inline test/test_token.js]: when create token

File: test/test_token.js [hunk 4]
```diff
@@ -73,7 +73,12 @@ describe('test create token', function () {
...
+        expect(create_original_data.isEqualTo(create_parse_function_data[0])).to.be.equal(true);
```
Comment (SheldonYS) [inline test/test_token.js]: Simplify to `expect(A).to.be.equal(B)` instead of `expect(A==B).to.be.equal(true)`

Comment (SheldonYS) [reply test/test_token.js]: If type is different, convert to string and then compared.

Comment (PKUzjw) [reply test/test_token.js]: I have tried in this way before, but it didn't work. 
image

Comment (PKUzjw) [reply test/test_token.js]: image

Comment (PKUzjw) [reply test/test_token.js]: I can change them to string and use expect(A).to.be.equal(B)

**Pull request merged**